src/baseline_bots/utils.py

```diff
@@ -236,7 +236,172 @@ def get_province_from_order(order):
     else:
         return order_tokens[0]
 
+def dipnet_to_daide_parsing(dipnet_style_order_strs: List[str], game: Game) -> List[str]:
+    """
+    Convert dipnet style single order to DAIDE style order. Needs game instance to determine the powers owning the units
+    * dipnet_style_order_strs: dipnet style list of orders to be converted to DAIDE
```

Comment:
This is not a huge deal, but lets try to do sphinx style docstrings,

so :param dipnet_style_order_strs: dipnet style list of orders to be converted to DAIDE

:return: ..., etc.